app/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,8 @@
     <string name="menu_page_other_languages">Change language</string>
     <string name="menu_page_find_in_page">Find in article</string>
     <string name="menu_page_add_to_watchlist">Add to Watchlist</string>
+    <string name="menu_page_talk">Talk</string>
+    <string name="menu_page_edit_history">Edit History</string>
```

Comment:
Should we have a lowercase of `history?

Reply:
She has kept it in Uppercase, probably like 'Add to Watchlist'

Reply:
Per our conversation, I changed this to lowercase @cooltey . So done.